gp-includes/things/original.php

```diff
@@ -440,5 +445,21 @@ public function after_delete() {
 
 		return true;
 	}
+
+	/**
+	 * Returns the appropriate cache group based on user caps
+	 *
+	 * @since 2.1.0
+	 *
+	 * @return string
+	 */
+	private function count_cache_group() {
+		$cache_group = self::$count_cache_group;
```

Comment:
Since your using `$this->` for the project id below, it should be consistent here instead of `SELF`.